assessment.js

```diff
@@ -0,0 +1,112 @@
+document.addEventListener("DOMContentLoaded", () => {
+    console.log("DOMContent is loaded")
+    selectTitle()
+    // displaySelectedFilm(film)                                                                                                                                                                                      select", selectTitle)
+    // let reviewInput = document.querySelector("#text-input")
+    // let form = document.querySelector("form")
+    // form.addEventListener("submit", (event) => {
+    //       let textInput = document.querySelector("#text-input")
+    //textInput.innerText = ""
+    let inputBtn = document.querySelector("#btn-input")
+    inputBtn.addEventListener("submit", (reviewSection))
```

Comment:
The right please to listen for the `submit` event is the form or any of its ancestors. Listening for `submit` event on the `button` will cause the event listener to never be fired because the button never 'feels' the `submit` as dictated by the event bubbling since the event generates in the form itself. This doesn't allow us to prevent the default and the page refreshes.